include/mender-utils.h

```diff
@@ -148,6 +148,14 @@ mender_err_t mender_utils_keystore_to_json(mender_keystore_t *keystore, cJSON **
  */
 mender_err_t mender_utils_keystore_set_item(mender_keystore_t *keystore, size_t index, char *name, char *value);
 
+/**
+ * @brief Function used to get index of item in key-store
+ * @param keystore Key-store
+ * @param name Name of the item
+ * @return Index of the found item in key-store (>= 0), Index not found returns -1
```

Comment:
Very minor: just for consistency of comment I suggest "Index of the item found in the key-store, -1 otherwise"

Reply:
Fixed